### PR TITLE
Splitting front end string with new symbol At-120

### PR DESCRIPTION
**Pull request by Neokrieger**
https://dfacademy.atlassian.net/jira/software/projects/AT/boards/4?issueParent=10138&selectedIssue=AT-120

UpdatedList

Updated with no bullet points:
UpdatedList

### CODE REVIEW

**Review by OllieVanD — APPROVED** [review]
LGMT

**Review by dearshrewdwit — COMMENTED** [review]
Can you hide the list item bullet point


https://www.w3schools.com/css/css_list.asp

File: src/components/Upload/Entry.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ changeCond(){
 
   render(){
     return(
-      <p id={`Text-${this.state.id}`}  style={{color: this.state.data.status === "Success" ? 'green' : 'red'}}>{this.state.data.status} {this.state.data.uploadType} {this.state.date} <button id={`Button-${this.state.id}`} onClick={() => this.changeCond()}>Check</button><ul id={`Text-${this.state.id}`} style={{fontSize: this.state.check ? 12 : 0}}>{this.state.errors.map(error => <li>{error}</li>)}</ul></p>
+      <p id={`Text-${this.state.id}`}  style={{color: this.state.data.status === "Success" ? 'green' : 'red'}}>{this.state.data.status} {this.state.data.uploadType} {this.state.date} <button id={`Button-${this.state.id}`} onClick={() => this.changeCond()}>Check</button><ul id={`Text-${this.state.id}`} style={{fontSize: this.state.check ? 12 : 0}}>{this.state.errors.map(error => <li style={{listStyle: "none"}}>{error}</li>)}</ul></p>
```
Comment (dearshrewdwit) [inline src/components/Upload/Entry.js:26]: Any component-specific style that's not calculated move to a `.css` file next to the component it's intended for

Like: 
Screenshot 2021-09-02 at 12 17 57

Comment (Neokrieger) [reply src/components/Upload/Entry.js:26]: @dearshrewdwit 
We are doing that in issue AT-115 . Just need this to go through before having the design updates

**Review by OllieVanD — APPROVED** [review]

**Review by dearshrewdwit — APPROVED** [review]

**Pull request merged**